.htaccess

```diff
@@ -21,6 +21,10 @@ DirectoryIndex index.php index.html index.htm
 # Override PHP settings that cannot be changed at runtime. See
 # settings.php and backdrop_environment_initialize() in includes/bootstrap.inc
 # for settings that can be changed at runtime.
+<IfModule mod_php7.c>
+  php_value max_input_vars                  10007
+</IfModule>
+# Same settings again for PHP 5 versions.
```

Comment:
With https://github.com/backdrop/backdrop/pull/3399 merged, this change should be removed, as it causes conflicts.